build/run_unit_group.sh

```diff
@@ -78,8 +78,8 @@ function test_group_broker_client_impl() {
   mvn_test -pl pulsar-broker -Dgroups='broker-impl'
 }
 
-function test_group_broker_jdk8() {
-  mvn_test -pl pulsar-broker -Dgroups='broker-jdk8' -Dpulsar.allocator.pooled=true
+function test_group_broker_jdk17() {
```

Comment:
We have only `ManagedLedgerCompressionTest` which tags `broker-jdk17`(and `broker` too).  I don't mind removing this `broker_jdk17` test as it is already covered in the broker test, but it might be useful in the future when we start supporting jdk18+, as a broker jdk base test, just like the original broker jdk8 test. 

```
@Test(groups = {"broker", "broker-jdk17"})
public class ManagedLedgerCompressionTest extends BrokerTestBase {
```